src/test/java/com/worksap/nlp/sudachi/OovProviderPluginTest.kt

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2022 Works Applications Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.worksap.nlp.sudachi
+
+import com.worksap.nlp.sudachi.dictionary.Grammar
+import com.worksap.nlp.sudachi.dictionary.POS
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFails
+import kotlin.test.assertIs
+
+class OovProviderPluginTest {
+  class FakeOovProvider : OovProviderPlugin() {
+    override fun provideOOV(
+        inputText: InputText?,
+        offset: Int,
+        otherWords: Long,
+        result: MutableList<LatticeNodeImpl>?
+    ): Int {
+      throw NotImplementedError("not implemented")
+    }
+
+    override fun setUp(grammar: Grammar?) {
+      val kind = settings.getString("posKind", "forbid")
```

Comment:
Is there any reason why it is `posKind`?

Reply:
completely accidential